test/functional/wallet_fundrawtransaction.py

```diff
@@ -975,6 +975,9 @@ def test_transaction_too_large(self):
         self.nodes[0].createwallet("large")
         wallet = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
         recipient = self.nodes[0].get_wallet_rpc("large")
+
+        # Force the wallet to bulk-generate the addresses we'll need
+        recipient.keypoolrefill(1500)
```

Comment:
nit: Could probably move this call a few lines down, immediately before the output generation loop so it's more obvious why 1500? (The `createrawtransaction` call doesn't make use of the keypool, so it's okay to prefill after).

Reply:
Updated!